src/main/java/pl/akademiaspecjalistowit/DocumentFlowManagementService/document/DocumentController.java

```diff
@@ -0,0 +1,22 @@
+package pl.akademiaspecjalistowit.DocumentFlowManagementService.document;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.akademiaspecjalistowit.DocumentFlowManagementService.document.dto.DocumentDto;
+import pl.akademiaspecjalistowit.DocumentFlowManagementService.document.service.DocumentService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/document")
```

Comment:
jak tworzymy API to fajnie by było jak by ścieżka się zaczynała z /api/document lub /api/documents -> to do rozkminy